common-util/AbiAndAddresses/serviceRegistryL2.js

```diff
@@ -0,0 +1,1892 @@
+export const SERVICE_REGISTRY_ADDRESS_BASE = '0x3C1fF68f5aa342D296d4DEe4Bb1cACCA912D95fE';
+
+export const SERVICE_REGISTRY_ABI = [
```

Comment:
tiny, i think name here may confuse in future if we have use `serviceRegistry` that's not L2?

@kupermind can you confirm if abis differ much?

Reply:
serviceRegistryL2 is different from serviceRegistry, but the required ABI is the same between the two. For the UI purposes it can be names without L2, I'm fine with that.

Reply:
I renamed for clarity